index.html

```diff
@@ -49,16 +52,9 @@ <h2><a href="http://twitter.com/speaker">@speaker</a></h2>
     <section class="slide compact">
         <h2>TODO: Please update slides:</h2>
         <ul>
-            <li>rename project at <mark>GitHub</mark> with snake-case convention</li>
```

Comment:
te informacje są instrukcją dla kolejnych prelegentów, aby wiedzieli jakie są kroki podczas tworzenia slajdów z oparciu o ten szablon

nie chciałbym tego usuwać

trzeba to przenieść w sensownej miejsce

@borowyalan zapronujesz coś?

Reply:
Faktycznie, gapa ze mnie.

Może po prostu sekcja w README?

## Updating the template to your needs:

 - rename project at <mark>GitHub</mark> with snake-case convention
- <mark>description</mark> and <mark>link</mark> on GitHub repository profile
- link to <mark>"Fork me on GitHub"</mark> at the bottom of the document</li>
- setup presentation title in
	- <mark>&lt;title&gt;</mark> tag on the top of document in <mark>&lt;head&gt;</mark> tag
	- <mark>README.md</mark> file
	- <mark>&lt;h1&gt;</mark> tag at <mark>&lt;header&gt;</mark> tag on the top of document

Reply:
Gites, mi pasuje :)